src/views/document/utils/field-viewers/MarkdownSection.vue

```diff
@@ -42,6 +42,14 @@
       return {
         visible: true
       };
+    },
+
+    computed: {
+      lang() {
+        const current_lang = this.$language.current;
+        const document_lang = this.document ? this.document.cooked.lang : null;
+        return document_lang === current_lang ? false : document_lang;
```

Comment:
```suggestion
        return document_lang === current_lang ? undefined : document_lang;
```

AFAIK, it's better to send `undefined` to HTML instead of `false` when we do not wan't to set it

Reply:
If it's similar to angular, and I do think it is, then false will set the attribute's value, while undefined will remove it.